style.css

```diff
@@ -0,0 +1,406 @@
+/*------------------------- Common Styles ----------------------------*/
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,300;0,400;0,600;0,700;1,300&display=swap');
+
+
+*, *:before, *:after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+:root {
+  --color-background: #ECEEF1;
+  --color-border: #c7c7c7;
+  --color-primary: #0077C5;
+}
+
+body {
+  background-color: var(--color-background);
+  font-size: 1.6rem;
+  font-family: 'Open Sans', sans-serif;
+}
+
+/*------------------------- Header Section ----------------------------*/
+.header-section {
+  display: flex;
+  height: 6rem;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 1rem 3rem;
+}
+
+.header-text {
+  display: flex;
+  flex-direction: column;
+  font-size: 1.4rem;
+  margin-right: 1rem;
+  text-align: right;
+}
+
+.header-text p {
+  color: grey;
+}
+
+.header-text a {
+  text-decoration: none;
+  color: var(--color-primary);
+}
+
+.header-button {
+  background-color: var(--color-primary);
+  border: none;
+  color: white;
+  padding: 1rem 2rem;
+  border-radius: 0.3rem;
+  font-size: 1.5rem;
+  text-decoration: none;
+}
+
+.header-button:focus {
+  outline: none;
+}
+
+.header-button i {
+  margin-right: 1rem;
+}
+
+/*------------------------- Main Section ----------------------------*/
+.main-section {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.main-top {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.logo-box {
+  width: 13rem;
+  height: auto;
+}
+
+
+
+.logo-box a {
+  text-decoration: none;
+}
+
+.logo-box img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
+
+.companies {
+  list-style: none;
+  display: flex;
+  justify-content: center;
+  text-align: center;
+  margin: 0 auto;
+  backface-visibility: hidden;
+}
+
+.companies div {
+  display: flex;
+  justify-content: center;
+}
+
+.companies div a {
+  text-decoration: none;
+}
+
+.mint-company, .quickbooks-company, .turbotax-company {
+  margin-right: 1.5rem;
+  margin-top: 0.5rem;
+}
+
+.mint-logo, .quickbooks-logo, .turbotax-logo {
+  width: 2.5rem;
+  height: 2.5rem;
+  margin-right: 0.2rem;
+}
+
+.mint-logo img, .quickbooks-logo img, .turbotax-logo img {
+  width: 100%;
+  height: 100%;
+  object-fit: contain;
+}
```

Comment:
The code on these lines can be grouped with the codes on the following lines as they do the same thing:

- lines 97-101